vm/vmrun.md

```diff
@@ -15,11 +18,35 @@ az vm run-command invoke -g myResourceGroup -n myVm \
   --scripts 'export FALCON_CLIENT_ID=123456789f1c4a0d9987a45123456789 && export FALCON_CLIENT_SECRET=ABCDEFGHtwfk6c0U4l72EsnjXxS1mH9123456789 && curl -L https://raw.githubusercontent.com/crowdstrike/falcon-scripts/main/bash/install/falcon-linux-install.sh | bash'
 ```
 
-## Using Run Command on a Windows instance
+To use the Run Command on multiple VMs, use the `az vm run-command invoke` command with the `--ids` parameter. The following example runs the Run Command on all VMs in a resource group named *myResourceGroup*.
+
+```azurecli
+az vm run-command invoke --ids $(az vm list -g myResourceGroup --query "[].id" -o tsv) \
+  --command-id RunShellScript \
+  --scripts 'export FALCON_CLIENT_ID=123456789f1c4a0d9987a45123456789 && export FALCON_CLIENT_SECRET=ABCDEFGHtwfk6c0U4l72EsnjXxS1mH9123456789 && curl -L https://raw.githubusercontent.com/crowdstrike/falcon-scripts/main/bash/install/falcon-linux-install.sh | bash'
+```
+
+## Using Run Command on Windows
 
 The following example runs the Run Command a Windows VM named *myVM* in a resource group named *myResourceGroup*. Replace the example resource group name, VM name, and script environment variables to run (https://raw.githubusercontent.com/CrowdStrike/falcon-scripts/main/powershell/install/falcon_windows_install.ps1) with your own information. For more Azure specific documentation using the Run Command, see [https://docs.microsoft.com/en-us/azure/virtual-machines/linux/run-command](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/run-command)
 
+> [!IMPORTANT]
+> Ensure the target VM's support TLS 1.2 or later. If you see errors related to TLS, try prepending `--scripts` with the following: `[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;`.
```

Comment:
Need an example or we can just add this to every command instead of calling it out.

It only modifies the current session so I don't see an issue with adding it to every command.

Reply:
I'll show an example.. @redhatrises said that if a customer is using TLS 1.3 this would be confusing/misleading.. so I'll just show an example

Reply:
Although come to think of it... @redhatrises wouldn't our script break if the environment only supported 1.3?

Reply:
Hopefully, we aren't hard coding TLS versions in the script even for a current session. If we are, that should be fixed.

Reply:
Our script hard codes 1.2

We are overthinking it. TLS 1.3 is not supported unless it's windows server 2022/win 11. `1.2` will be here for awhile and is supported on every windows OS since vista. The current issue is 2016 machines don't use 1.2 by default hence the need for the workaround.